CMakeLists.txt

```diff
@@ -1,56 +1,78 @@
-cmake_minimum_required(VERSION 3.0)
+cmake_minimum_required(VERSION 3.0.0)
 
-# define the project
-project(nlohmann_json VERSION 2.1.1 LANGUAGES CXX)
+##
+## PROJECT
+## name and version
+##
+project(nlohmann_json VERSION 2.1.1)
```

Comment:
Thanks for checking back! Just out of coincidence: would it help to explicitly enable C++, because we do not use C at all?